From 6516ac6993ff14f67317e0b75d028477364b7e5b Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Wed, 5 Apr 2017 09:36:58 +0300
Subject: [PATCH 1565/2241] fix: dts: MACCHIATObin: set mac3 to SGMII 2.5G mode

According to MACCHIATObin documentation, mac3 should
be configured to SGMII 2.5 mode:
http://wiki.macchiatobin.net/tiki-index.php?page=Block+diagram

Change-Id: I7e4f1ca7d8480214fe2e84a6dd23f2f0baffbbc2
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/38519
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index 4386ef0..324943e 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -215,7 +215,8 @@
 				};
 				emac3_1: mac3 {
 					phy-mode = "sgmii"; /* lane-5 */
-					phys = <&comphy1 5 COMPHY_SGMII2>;
+					phy-speed = <2500>;
+					phys = <&comphy1 5 COMPHY_HS_SGMII2>;
 					phy-names = "comphy";
 				};
 			};
-- 
2.7.4

